client/src/components/landing/components/faq.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Col, Row } from '@freecodecamp/react-bootstrap';
+import { useTranslation } from 'react-i18next';
+import { Spacer } from '../../helpers';
+import BigCallToAction from './big-call-to-action';
+
+interface FaqItem {
+  question: string;
+  answer: string[];
+}
+
+const Faq = (): JSX.Element => {
+  const { t } = useTranslation();
+  const faqItems: FaqItem[] = t('landing.faqs', { returnObjects: true });
+
+  return (
+    <Row className='landing-faq'>
+      <Col sm={8} smOffset={2} xs={10} xsOffset={1}>
+        <h1 className='big-heading'>{t('landing.faq')}</h1>
+        <Spacer size='medium' />
+        {faqItems.map((faq, i) => (
+          <div data-test-label='landing-page-faq' key={i}>
+            <h3>{faq.question}</h3>
+            {faq.answer.map((answer, i) => (
+              <p key={i}>{answer}</p>
+            ))}
+            <Spacer size='small' />
+            <Spacer size='small' />
+          </div>
+        ))}
+        <h3>{t('landing.happy-coding')}</h3>
+        <Spacer size='medium' />
+        <BigCallToAction />
+        <Spacer size='large' />
+      </Col>
+    </Row>
+  );
```

Comment:
We don't need the row element, the className. and if you are okay with making the answer one string, we can remove extra map higher function.

```suggestion
  return (
    <Col sm={8} smOffset={2} xs={10} xsOffset={1}>
      <h2 className='big-heading'>{t('landing.faq')}</h2>
      <Spacer size='medium' />
      {faqItems.map((faq, i) => (
        <div data-test-label='landing-page-faq' key={i}>
          <p className='faq-question'>{faq.question}</p>
          <p>{faq.answer}</p>
          <Spacer size='xSmall' />
        </div>
      ))}
      <p>{t('learn.read-this.p12')}</p>
      <Spacer size='medium' />
      <BigCallToAction />
      <Spacer size='large' />
    </Col>
  );
```

About the headers, they indicate that these are a section for the screen readers, but the questions aren't sections, they are part of FAQ section, so to not confuse screen reader, we can use `p` elements, because the `details` don't give a good look and will require more work.

You can add the wanted styles in landing.css file https://github.com/freeCodeCamp/freeCodeCamp/blob/f5070cc68fa4caaaf668b65f4b05eb2d4bff055b/client/src/components/landing/landing.css#L1

```css
.faq-question {
  font-weight: bold;
  font-size: 1.25rem;
}
```